### ISSUE TITLE
Need an option to hide contacts in default search view.

### ISSUE THREAD

**Issue by teja2495** [BranchMetrics/sesame_issues#251]
Hi, 

The default search screen in sesame shows a bunch of apps and a bunch of contacts. App suggestions are fine. But contacts are super annoying. It's showing me the list of contacts that I never spoke in years. At least 80% of the screen is filled with contacts. It's okay if you fill it with recently searched / recently contacted contacts, but putting every random contact in that huge list is not good in my opinion. Can you please give us an option to disable contacts in default search view and only show them when we are actually making a search?

Thanks
Teja

**Comment by papadi** [comment]
True. I get random contacts which I haven't talked to in years. Why is that?

**Comment by papadi** [comment]
I guess that one way to fix this is to go to contact settings and toggle them all off. But then you will just get random options from WhatsApp.
